### PR TITLE
Schedule formulas execution through CRON jobs. Issue https://github.c…

### PR DESCRIPTION
**Pull request by joaomsilvadev**
…om/ZupIT/ritchie-cli/issues/641

# Pull Request

## What I did

Schedule formulas execution through CRON jobs for Linux systems as requested in the issue https://github.com/ZupIT/ritchie-cli/issues/641

## How I did it

Creating a new formula cron schedule

## How to verify it

Run the command "rit cron schedule" and follow the instructions.

## What kind of change does this PR make

- [ ] Major
- [x] Minor
- [ ] Patch

## Description for the changelog

Schedule formulas execution through CRON jobs for Linux systems.

### CODE REVIEW

**Comment by GuillaumeFalourd** [comment]
Hi @joaomsilvadev, and thank you for contributing to Ritchie with your formula! 🚀 

As the formula you implemented isn't support on Windows, I suggest to remove the `formula.bat` file. Otherwise, users running this formula on this OS will execute the *Hello World* template.

**Comment by joaomsilvadev** [comment]
> rmula you implemented isn't support on Windows, I suggest to remove the `formula.bat` file. Otherwise, users running this formula on this OS will execute the _Hello World_ template.
> 
> What do you think?

I'll do it!
Thanks for the quick response!

**Comment by GuillaumeFalourd** [comment]
> I'll do it!
> Thanks for the quick response!

I forgot other files that need to be removed or updated as well due to incompatibility with windows:
- `main.bat` file.
- `build.bat` file.

And I also suggest to update the formula `README.md` file to make it easier for people to understand through the [formula portal](https://formulas.ritchiecli.io/list). Nothing to sophisticated, you can follow [this formula README file](https://github.com/ZupIT/ritchie-formulas/tree/master/github/create/repo) as reference.

File: cron/schedule/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+## Command
+
+```bash
+rit cron schedule
+```
+
+## Requirements
+
+## Demonstration
```
Comment (GuillaumeFalourd) [inline cron/schedule/README.md:13]: Could update this file please? 
[suggestion](https://github.com/ZupIT/ritchie-formulas/pull/367#issuecomment-850640293)

File: cron/schedule/build.bat [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  rename %ENTRY_POINT% %BINARY_NAME%
+
+:CP_DOCKER
+  cd ..
+  copy Dockerfile %BIN_FOLDER%
+  copy set_umask.sh %BIN_FOLDER%
+
+:EXIT
+  ENDLOCAL
+  exit /b
```
Comment (GuillaumeFalourd) [inline cron/schedule/build.bat:31]: Could can remove this file as it is incompatible with Windows OS.
[suggestion](https://github.com/ZupIT/ritchie-formulas/pull/367#issuecomment-850640293)

File: cron/schedule/src/main.bat [hunk 3]
```diff
@@ -0,0 +1,3 @@
+@echo off
+
+call windows\formula\formula.bat runFormula
```
Comment (GuillaumeFalourd) [inline cron/schedule/src/main.bat:3]: Could can remove this file as it is incompatible with Windows OS.
[suggestion](https://github.com/ZupIT/ritchie-formulas/pull/367#issuecomment-850640293)

File: cron/schedule/src/windows/formula/formula.bat [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  )
+
+  echo Today, I want to automate %RIT_INPUT_LIST%.
+  echo My secret is %RIT_INPUT_PASSWORD%.
+
+  goto exit
+
+:exit
+  ENDLOCAL
+  exit /b 0
```
Comment (GuillaumeFalourd) [inline cron/schedule/src/windows/formula/formula.bat:24]: Could can remove this file as it is incompatible with Windows OS.
[suggestion](https://github.com/ZupIT/ritchie-formulas/pull/367#issuecomment-850640293)

**Comment by joaomsilvadev** [comment]
I made more changes and added list and remove cron jobs functionality as well.

**Review by GuillaumeFalourd — APPROVED** [review]
Thank you for updating your PR @joaomsilvadev ! 💯 🚀

**Comment by GuillaumeFalourd** [comment]
It seems some files didn't pass the Pre-Commit execution @joaomsilvadev (could be related to whitespaces or similar issues). You can check what happened here: https://github.com/ZupIT/ritchie-formulas/pull/367/checks?check_run_id=2805843591

**Comment by joaomsilvadev** [comment]
Could someone please help me solve these problems?

**Comment by GuillaumeFalourd** [comment]
@joaomsilvadev : you can use the https://github.com/ZupIT/ritchie-formulas/blob/master/pre-commit.sh or https://github.com/ZupIT/ritchie-formulas/blob/master/pre-commit.bat files locally to check your files format before pushing your code. It should return the same erros we observed when performing [those checks](https://github.com/ZupIT/ritchie-formulas/pull/367/checks?check_run_id=2806264337) on the PR

**Comment by joaomsilvadev** [comment]
Why am I having these issues below? My code is written in Python. There's nothing from Golang

```

 go fmt...................................................................Failed

- hook id: go-fmt
- exit code: 127

/root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found

go imports...............................................................Failed
- hook id: go-imports
- exit code: 127

/root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
/root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
```

**Comment by GuillaumeFalourd** [comment]
> Why am I having these issues below? My code is written in Python. There's nothing from Golang
> 
> ```
> 
>  go fmt...................................................................Failed
> 
> - hook id: go-fmt
> - exit code: 127
> 
> /root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-fmt.sh: line 8: gofmt: command not found
> 
> go imports...............................................................Failed
> - hook id: go-imports
> - exit code: 127
> 
> /root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
> /root/.cache/pre-commit/repoxnr37caq/run-go-imports.sh: line 8: goimports: command not found
> ```

We'll investigate and I'll give you a return ASAP 👍🏼 
Have a nice week-end until then 😉

**Comment by fabianofernandeszup** [comment]
@joaomsilvadev  We've updated the .pre-commit-config.yaml file on the master branch with the fix for this issue.
Update your branch with master and your PR should pass.
**Thanks for the contribution.**

**Comment by joaomsilvadev** [comment]
Hi
Could you approve my PR please?

**Review by GuillaumeFalourd — APPROVED** [review]

**Comment by GuillaumeFalourd** [comment]
Sorry for the delay @joaomsilvadev , I missed your last message after @fabianofernandeszup's return.
All good to go! 👍🏼

**Pull request merged**